frontend/src/scenes/insights/insightLogic.tsx

```diff
@@ -621,6 +623,11 @@ export const insightLogic: LogicWrapper<insightLogicType> = kea<insightLogicType
             })
             lemonToast.success(`Insight ${feedback}`)
         },
+        trackInsightViewed: () => {
+            if (values.insight.id && values.currentTeamId) {
+                void api.create(`api/environments/${values.currentTeamId}/insights/${values.insight.id}/viewed`)
```

Comment:
we will make this call for every insight right?
would be better if we can just figure out in bulk the last viewed for all insights in one API call.
Also where do we store the last viewed? maybe we can store it on the insight itself so its trivial to get back (just thinking out loud haven't thought through the implications/if there's a better solution)